.github/ci_script/combine_log.py

```diff
@@ -0,0 +1,45 @@
+import time
+import sys
+import os
+'''
+  Get info.
+    output_path: the target file that you want to combine sub log with.
+    list_path: the list of sub log name. When it is updated, the correspondding file will be add to output tail.
+    list_dir_path: the dir path where sub logs stored.
+    status_path: the path of status file. When status file is written to "success" or "fail", exit script.
+'''
```

Comment:
use argparse, you may use chatgpt to help rewrite this scripts.

Reply:
Done